recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ test:
 
     # Check libraries.
     - test -f "${PREFIX}/lib/libyaml${SHLIB_EXT}"     # [not win]
-    - test -f "${PREFIX}/lib/libyaml-0.2${SHLIB_EXT}"  # [osx]
+    - test -f "${PREFIX}/lib/libyaml.0.2${SHLIB_EXT}"  # [osx]
```

Comment:
This change is wrong. See the previous build:
<img width="346" alt="Screen Shot 2021-12-15 at 11 42 44 AM" src="https://user-images.githubusercontent.com/5296416/146237723-c9328837-a117-423d-bfc9-19d7584dfcfb.png">

Reply:
That's what make was giving, cmake uses the other format.  I think both are valid.  🤷